src/printer.js

```diff
@@ -232,9 +232,22 @@ function genericPrintNoParens(path, options, print, args) {
         path.call(print, "right")
       ]);
     case "MemberExpression": {
+      const parent = path.getParentNode();
+      const shouldInline =
+        n.computed ||
```

Comment:
Yes, the `[` is inline, but the content inside can break.

```js
somethingLong[
  somethingLong
]
```

looks better than

```js
somethingLong
  [somethingLong]
```